<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindWidget</class>
 <widget class="QWidget" name="FindWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find / Replace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxMatchCase">
       <property name="toolTip">
        <string>Determines whether the search should be case sensitive or not.</string>
       </property>
       <property name="text">
        <string>Match case</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxRegEx">
       <property name="toolTip">
        <string>If this is checked, then the search term is interpreted as regular expression.</string>
       </property>
       <property name="text">
        <string>RegEx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxSearchTitles">
       <property name="toolTip">
        <string>Include chapter and scene titles in the search.</string>
       </property>
       <property name="text">
        <string>Search titles</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxScope">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Determines search scope.&lt;/p&gt;
&lt;dl&gt;
  &lt;dt&gt;Scene&lt;/dt&gt;
  &lt;dd&gt;Searches currently open scene only.&lt;/dd&gt;
  &lt;dt&gt;Chapter&lt;/dt&gt;
  &lt;dd&gt;Searches the chapter that includes the currently open scene.&lt;/dd&gt;
  &lt;dt&gt;Project&lt;/dt&gt;
  &lt;dd&gt;Searches the entire project.&lt;/dd&gt;
&lt;/dl&gt;
&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>Scene</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chapter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Project</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFind">
       <property name="text">
        <string>Find:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditFind</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditFind">
       <property name="placeholderText">
        <string>Search term</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonSearch">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Start the search</string>
         </property>
         <property name="text">
          <string notr="true">üîç</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPrev">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Go to previous result</string>
         </property>
         <property name="text">
          <string notr="true">‚óÄ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonNext">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Go to next result</string>
         </property>
         <property name="text">
          <string notr="true">‚ñ∂</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonExclude">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Exclude current result from replacement candidates</string>
       </property>
       <property name="text">
        <string>Exclude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelReplace">
       <property name="text">
        <string>Replace:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditReplace</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditReplace">
       <property name="placeholderText">
        <string>Replacement</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonReplace">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Replace the current result</string>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButtonReplaceAll">
       <property name="toolTip">
        <string>Replace all results that have not been excluded</string>
       </property>
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxMatchCase</tabstop>
  <tabstop>checkBoxRegEx</tabstop>
  <tabstop>checkBoxSearchTitles</tabstop>
  <tabstop>comboBoxScope</tabstop>
  <tabstop>lineEditFind</tabstop>
  <tabstop>lineEditReplace</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonPrev</tabstop>
  <tabstop>pushButtonNext</tabstop>
  <tabstop>pushButtonExclude</tabstop>
  <tabstop>pushButtonReplace</tabstop>
  <tabstop>pushButtonReplaceAll</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditFind</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonSearch</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
